nextmove/src/main/java/no/difi/meldingsutveksling/arkivmelding/ArkivmeldingUtil.java

```diff
@@ -0,0 +1,35 @@
+package no.difi.meldingsutveksling.arkivmelding;
+
+import com.google.common.collect.Lists;
+import no.arkivverket.standarder.noark5.arkivmelding.Arkivmelding;
+import no.arkivverket.standarder.noark5.arkivmelding.Dokumentbeskrivelse;
+import no.arkivverket.standarder.noark5.arkivmelding.Journalpost;
+import no.arkivverket.standarder.noark5.arkivmelding.Saksmappe;
+
+import java.util.List;
+
+public class ArkivmeldingUtil {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1118)